build-system/babel-config/minified-config.js

```diff
@@ -72,14 +72,13 @@ function getMinifiedConfig() {
     {
       bugfixes: true,
       modules: false,
-      targets: {esmodules: true},
+      targets: argv.esm ? {esmodules: true} : {browsers: ['Last 2 versions']},
```

Comment:
Last 2 versions likely performs [more transpilation](https://jamie.build/last-2-versions) than necessary. I think you can put just `ie11` as the browser target

Reply:
Updated this.